com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/FrameSettings.cs

```diff
@@ -105,6 +105,9 @@ public enum FrameSettingsField
         /// <summary>When enabled, Cameras using these Frame Settings use Alpha To Mask. Activate MSAA to access this option.</summary>
         [FrameSettingsField(0, displayedName: "Alpha To Mask", positiveDependencies: new[] { MSAA }, customOrderInGroup: 3, tooltip: "When enabled, Cameras using these Frame Settings use Alpha To Mask. Activate MSAA to access this option.")]
         AlphaToMask = 56,
+        /// <summary>When enabled, The emissive part of the materials is rendered as a forward pass for the deferred pipeline.</summary>
+        [FrameSettingsField(0, displayedName: "Emissive as Forward", positiveDependencies: new[] { LitShaderMode }, customOrderInGroup: 3, tooltip: "When enabled, The emissive part of the materials is rendered as a forward pass for the deferred pipeline.")]
+        EmissiveAsForward = 57,
```

Comment:
Can you name it "LitEmissiveAsForward" so we understand it is only for the Lit shader. Thanks